### PR TITLE
INGK-913 Test ethernet only on Everest

### PR DESCRIPTION
**Pull request by pzinemanas**
### Description

Please include a summary of the change and which issue is fixed. 

Fixes # INGK-913

### Type of change

- [x] Test ethernet on Everest instead of Capitan.

### Others

- [x] Set fix version field in the Jira issue.

### CODE REVIEW

File: tests/test_servo.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def create_monitoring(connect_to_slave, pytestconfig):
     servo, net = connect_to_slave
     servo.monitoring_disable()
     servo.monitoring_remove_all_mapped_registers()
-    registers_key = ["CL_POS_SET_POINT_VALUE"]
+    registers_key = ["CL_CUR_D_REF_VALUE"]
```
Comment (MartinALL91) [inline tests/test_servo.py:37]: Why is this change needed?

File: tests/config.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-    "dictionary": "//awe-srv-max-prd/distext/products/CAP-XCR/firmware/2.4.0/cap-xcr-c_eth_2.4.0.xdf",
-    "ip": "192.168.2.11",
+    "dictionary": "//awe-srv-max-prd/distext/products/EVE-XCR/firmware/2.4.0/eve-xcr-c_eth_2.4.0.xdf",
+    "ip": "192.168.2.10",
     "port": 1061,
     "save_config_file": "ethernet-config.xcf",
-    "load_config_file": "//azr-srv-ingfs1/dist/setups/setup_cap_can/1.1.0/config.xml",
-    "fw_file": "//awe-srv-max-prd/distext/products/CAP-XCR/firmware/2.4.0/cap-xcr-c_2.4.0.lfu"
+    "load_config_file": "//azr-srv-ingfs1/dist/setups/setup_eve_can/1.2.0/config.xml",
+    "fw_file": "//awe-srv-max-prd/distext/products/EVE-XCR/firmware/2.0.0/eve-xcr-c_2.4.0.sfu"
```
Comment (GullinBustin) [inline tests/config.json:9]: FW file path is wrong version folder should be 2.4.0

**Review by GullinBustin — CHANGES_REQUESTED** [review]

File: tests/test_servo.py (hunk 1 above)
Comment (pzinemanas) [reply tests/test_servo.py:37]: Beacuse CL_POS_SET_POINT_VALUE is CYCLIC_RX so it could not be mapped in monitoring. Not sure why it worked in Capitan.

File: tests/config.json (hunk 2 above)
Comment (pzinemanas) [reply tests/config.json]: Fixed [0795c44](https://github.com/ingeniamc/ingenialink-python/pull/424/commits/0795c44d91d3fe03d37c3995e047f41baad68f42)[0795c44](https://github.com/ingeniamc/ingenialink-python/pull/424/commits/0795c44d91d3fe03d37c3995e047f41baad68f42)

**Comment by polfeliu** [comment]
https://novantamotion.atlassian.net/browse/INGK-914

**Review by polfeliu — APPROVED** [review]
